src/api/types.ts

```diff
@@ -1105,7 +1168,7 @@ export type HostValidationId =
   | 'container-images-available'
   | 'lso-requirements-satisfied'
   | 'ocs-requirements-satisfied'
-  | 'sufficient-installation-disk-speed'
+  | 'sufficient-or-unknown-installation-disk-speed'
```

Comment:
This key needs to be updated in `config/constants.ts`

Reply:
fixed